src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -9,6 +9,7 @@
     <IsPackable Condition="'$(IsPackable)' != ''">true</IsPackable>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <TieredCompilation>false</TieredCompilation>
+    <NoWarn>NU5104</NoWarn>
```

Comment:
This warning is about dependency on preview packages: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104 

We do have few dependencies on preview packages. So added `NoWarn` at project level